tests/comdb2sys.test/testapihistory

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+a_dbn=$1
+
+cdb2sql ${CDB2_OPTIONS} $a_dbn default "SELECT CASE WHEN length(host) > 0 THEN 'hostname' ELSE '' END AS host, task, api_driver_name, api_driver_version FROM comdb2_api_history WHERE task='cdb2sql'" > testapihistory.out
```

Comment:
```
16:10:46> Failed systable comdb2_api_history test
16:10:46> diff /dedicated/db/comdb2sys_pagesize_generated.test/testapihistory.out /dedicated/db/comdb2sys_pagesize_generated.test/testapihistory.expected
```

I think the test needs to query against a specific node (ie direct-cpu), so that it reliably passes.